core/templates/dev/head/pages/admin/admin.html

```diff
@@ -429,10 +68,24 @@
 
     {% include 'components/forms/form_builder_templates.html' %}
     {% include 'pages/footer_js_libs.html' %}
+    {% include 'pages/admin/admin_navbar_directive.html' %}
+    {% include 'pages/admin/activities_tab/admin_dev_mode_activities_tab_directive.html' %}
+    {% include 'pages/admin/activities_tab/admin_prod_mode_activities_tab_directive.html' %}
+    {% include 'pages/admin/config_tab/admin_config_tab_directive.html' %}
+    {% include 'pages/admin/jobs_tab/admin_jobs_tab_directive.html' %}
+    {% include 'pages/admin/misc_tab/admin_misc_tab_directive.html' %}
 
     <script src="{{TEMPLATE_DIR_PREFIX}}/app.js"></script>
     <script src="{{TEMPLATE_DIR_PREFIX}}/pages/Base.js"></script>
     <script src="{{TEMPLATE_DIR_PREFIX}}/pages/admin/Admin.js"></script>
+    <script src="{{TEMPLATE_DIR_PREFIX}}/pages/admin/AdminNavbarDirective.js"></script>
+    <script src="{{TEMPLATE_DIR_PREFIX}}/pages/admin/AdminRouterService.js"></script>
+    <script src="{{TEMPLATE_DIR_PREFIX}}/pages/admin/AdminTaskManagerService.js"></script>
+    <script src="{{TEMPLATE_DIR_PREFIX}}/pages/admin/activities_tab/AdminDevModeActivitiesTabDirective.js"></script>
```

Comment:
For the activities tab, could we only load the relevant templates and JS files depending on whether it's in dev/prod mode? That way, in prod mode, there is absolutely no chance of a reload comment being sent from this page to the backend.